src/main/helm/helm-service/get-helm-release-resources/call-for-kube-resources-by-manifest/exec-file-with-input/exec-file-with-input.injectable.ts

```diff
@@ -26,7 +26,9 @@ const execFileWithInputInjectable = getInjectable({
         let execution: ChildProcess;
 
         try {
-          execution = execFile(filePath, commandArguments);
+          execution = execFile(filePath, commandArguments, {
+            maxBuffer: 8 * 1024 * 1024 * 1024, // 8 MiB
```

Comment:
how do we know this is enough?

Reply:
I guess we don't. The default is 200KiB so this is about 20x the size.